src/main/java/com/agilefaq/codedoctorsample/App.java

```diff
@@ -12,7 +16,220 @@ public static int calc2(int a1, int a2) {
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        return a1 * a2;
+    }
+
+    public static int multiply5(int a1, int a2) {
+
+        Connection conn = null;
+        try {
+            conn = DriverManager.getConnection("jdbc:mysql://localhost/test?" +
+                    "user=steve&password=blue");
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        String uname = "steve";
+        String password = "blue";
+        try {
+            conn = DriverManager.getConnection("jdbc:mysql://localhost/test?" +
+                    "user=" + uname + "&password=" + password);
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+
+        java.net.PasswordAuthentication pa = new java.net.PasswordAuthentication("userName", "1234".toCharArray());
+
+
+        String a = null;
+        System.out.println(a.charAt(0));
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        return a1 * a2;
+    }
+
+    public static int multiply6(int a1, int a2) {
+
+        Connection conn = null;
+        try {
+            conn = DriverManager.getConnection("jdbc:mysql://localhost/test?" +
+                    "user=steve&password=blue");
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        String uname = "steve";
+        String password = "blue";
+        try {
+            conn = DriverManager.getConnection("jdbc:mysql://localhost/test?" +
+                    "user=" + uname + "&password=" + password);
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+
+        java.net.PasswordAuthentication pa = new java.net.PasswordAuthentication("userName", "1234".toCharArray());
+
+
+        String a = null;
+        System.out.println(a.charAt(0));
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "a1". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1226)